### ISSUE TITLE
1.15.2 ?

### ISSUE THREAD

**Issue by samuelh2005** [SquidDev-CC/ccjam-2016#254]
Says it in the title. So i've been looking for a more up to date Open Peripherals clone and i have found this but sadly its not updated to 1.15.2. I am not forcing you do do it now but when you can please do! I want this version because some CC programs i used require Open Peripherals.

**Comment by fatboychummy** [comment]
If your program requires Open Peripherals, Plethora will not work as a replacement.

Plethora provides different peripheral methods, and won't be compatible.  It should be easy enough to fix the program to be compatible, but it won't be a perfect "replacement".

**Comment by noncom** [comment]
Yes, it's different, but we need it for 1.15.2 anyways! Without this library CC-Tweaked is pretty much a thing in itself except for maybe the turtles, but their scope is totally different.

**Comment by noncom** [comment]
Maybe needs a rename to 1.16 :D

**Comment by sirati** [comment]
I am interested in helping getting this to modern minecraft

**Comment by Lupus590** [comment]
There are a few forks where people have tried to port the mod to more modern MC. Those wanting to help should probably look through the fork list (tree?) to continue other peoples work.

Most of the attempts are abandoned so just fork them yourself.

**Comment by sirati** [comment]
Will do thanks
  

  

  

  
  
  
  
  
>
